### DISCUSSION TITLE
Birthday 22 · Cup Air

### DISCUSSION THREAD

**Discussion by ACupofAir** [ACupofAir/ACupofAir.github.io#16] [Announcements]
https://acupofair.github.io/posts/life_is_poem/birth22/ 

 
就这样22岁了。这是一个在法律上可以支撑起家庭的年纪，兄弟，我希望你真的可以。

**Comment by thisisbadBao** [comment]
迟来的生日快乐🎉

**Comment by LeiBuDao** [comment]
迟了两个月的生日快乐🎉🎉🎉
